deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -23,6 +23,12 @@ spec:
       - image: gluster/gluster-centos:latest
         imagePullPolicy: IfNotPresent
         name: glusterfs
+        env:
+        - name: SYSTEMD_IGNORE_CHROOT
+          value: "1"
+        command:
+        - /usr/lib/systemd/systemd
+        - --system
```

Comment:
@humblec where is this insight from? At least in the regular manpages and docs I found, there is no such statement. Not saying you are wrong, but I'm curious to see references.